control/grpc.py

```diff
@@ -124,474 +126,452 @@ def _alloc_cluster(self) -> str:
         )
         return name
 
-    def create_bdev_safe(self, request, context=None):
+    def create_bdev(self, request, context=None):
```

Comment:
can we not remove the safe method?
It creates lots of indentation changes that make review hard and it is already there

Reply:
I don't really care but you were the one who wanted to do that. I sent you the "git diff" output ignoring white spaces so there is not that much to review there.